doc/win32-usb-troubleshoot.md

```diff
@@ -6,27 +6,66 @@ While this makes it easier to develop nRF Connect Desktop applications, it might
 
-## Windows 7 right after installation
+## Connecting a USB device into Windows 7 for the first time
 
-A `LIBUSB_ERROR_NOT_FOUND` error shows up when performing the following actions in this exact order:
+A `LIBUSB_ERROR_NOT_FOUND` error shows up when connecting an nRF USB device into a Windows 7 computer for the first time. In particular, this sequence of actions (in this exact order) reproduces this scenario:
 
 - Run the nRF Connect Desktop installer.
   - This includes the *rules* for the right drivers, but not the actual link between a USB device and its driver.
 - Launch an nRF Connect Desktop application.
-- Plug an nRF52840 devkit into a USB port.
+- Plug an nRF52840 device into a USB port.
 - Let Windows 7 attach the right drivers.
   - This uses the *rules* already installed for actually *binding* a driver to the newly-connected nRF USB device.
   - A yellow bubble appears from the system tray, indicating driver installation, like this: 
   ![screenshot](win32-drivers-installing.png)
 
 - A `LIBUSB_ERROR_NOT_FOUND` is shown on the log of the nRF Connect Desktop application.
-- After a couple of minutes, Windows 7 finishes binding the libusb driver to the nRF USB device.
+- After a time (that varies between a few seconds and a couple of minutes), Windows 7 finishes binding the libusb driver to the nRF USB device.
   - A yellow bubble appears from the system tray, indicating drivers are ready, like this:
   ![screenshot](win32-drivers-ready.png)
 
+Please note that a USB device is "new" for Windows if the combination of its USB vendor ID, product ID, revision, and serial number have never been seen before. In other words: changing the firmware in an nRF device, or setting it into bootloader mode will make it appear as "new" for Windows.
+
+Therefore, this error might happen the first time that an nRF USB device is connected, the first time it enters bootloader mode, and the first time it is loaded with application-specific USB-capable firmware.
+
 #### Workaround
 
 - Close all programs and processes that might be using nRF USB devices (including nRF Connect Desktop).
 - Connect the nRF USB device(s).
 - **Wait** until Windows 7 has finished binding the right drivers to the USB device(s).
 - Launch nRF Connect Desktop.
+
+If the nRF USB device enters bootloader mode, or is loaded with new firmware, you may need to repeat this workaround.
+
+## USB devices without an expected interface
+
+A `LIBUSB_ERROR_NOT_FOUND` error shows up when connecting an nRF USB device **without** a DFU trigger interface.
+
+In order to check if this is also your scenario, launch the Windows Device Manager, set the view to "Devices by Connection", and find the part of the USB device tree with the nRF USB Device.
+
+The nRF USB device shall appear as an "USB Composite Device" and, beneath that, each USB interface appears as a device in the tree. nRF Connect Desktop *expects* a "nRF Connect DFU trigger" interface to be present, like so:
```

Comment:
No, this time I'm pretty sure that "an USB" is the right grammar.

Reply:
I have to respectfully hold my opposition. A USB, because pronunciation of USB is: `juː-ɛs-biː`. There is also an occurrence of "nRF" which works the other way around.

Reply:
I just found that there actually exists a dedicated dictionary for this online: [a-or-an.com](https://www.a-or-an.com/a_an/usb) :smile:

Reply:
Ha, it seems that I've been pronouncing `iuː-ɛs-biː` wrong all this time. :neutral_face: